common/pom.xml

```diff
@@ -118,6 +118,11 @@
             <version>${janino-version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>edu.ucar</groupId>
+            <artifactId>cdm-core</artifactId>
+            <scope>compile</scope>
```

Comment:
Please do not specify `compile` here as it follows the same scope of its parent

Reply:
Removed scope